composer.json

```diff
@@ -60,7 +60,7 @@
         "monolog/monolog": ">=1.11.0",
         "vimeo/psalm": "^4",
         "php-http/cache-plugin": "^1.7",
-        "wmde/psr-log-test-doubles": "^2"
+        "jeroen/psr-log-test-doubles": "^2.1 || ^3"
```

Comment:
This is the new location where I maintain the library. Version 3 is needed to support the new `psr/log` versions.